jsapp/js/account/add-ons/addOnList.component.tsx

```diff
@@ -60,118 +60,155 @@ const AddOnList = (props: {
+    addOns: SubscriptionInfo[],
+    oneTimeAddOns: OneTimeAddOn[],
+    activeStatus: string,
+    available: boolean,
+    label: string,
+    badgeLabel: string,
+    color: BadgeColor
+  ) => (
     <table className={styles.table}>
-      <caption className={styles.caption}>
-        <label className={styles.header}>{t('available add-ons')}</label>
-        <p>
-          {t(
-            `Add-ons can be added to your Community plan to increase your usage limits. If you are approaching or
-            have reached the usage limits included with your plan, increase your limits with add-ons to continue
-            data collection.`
-          )}
-        </p>
+      <caption className={`${styles.caption} ${styles.purchasedAddOns}`}>
+        <label className={styles.header}>{label}</label>
       </caption>
       <tbody>
-        {addOnProducts.map((product) => {
-          if (isOneTimeAddonProduct(product) && props.organization) {
+        {addOns.map((product) => {
+          if (product.status === activeStatus) {
             return (
-              <OneTimeAddOnRow
-                key={product.id}
-                product={product}
-                isDisabled={props.isBusy}
-                organization={props.organization}
-              />
+              <tr className={styles.row} key={product.id}>
+                <td className={styles.product}>
+                  <span className={styles.productName}>
+                    {product.items[0].price.product.name}
+                  </span>
+                  <Badge color={color} size={'s'} label={badgeLabel} />
+                  <p className={styles.description}>
+                    {`Added on ${formatDate(product.created)}`}
+                  </p>
+                </td>
+                <td className={styles.activePrice}>
+                  {product.items[0].price.human_readable_price
+                    .replace('USD/month', '')
+                    .replace('USD/year', '')}
+                </td>
+              </tr>
+            );
+          }
+          return null;
+        })}
+        {oneTimeAddOns.map((oneTimeAddOn: OneTimeAddOn) => {
+          if (oneTimeAddOn.is_available === available) {
+            return (
+              <tr className={styles.row} key={oneTimeAddOn.id}>
+                <td className={styles.product}>
+                  <span className={styles.productName}>
+                    {oneTimeAddOneProducts[0].name +
+                      ' x ' +
+                      oneTimeAddOn.quantity}
+                  </span>
+                  <Badge color={color} size={'s'} label={badgeLabel} />
+                  <p className={styles.description}>
+                    {`Added on ${formatDate(oneTimeAddOn.created)}`}
+                  </p>
+                </td>
+                <td className={styles.activePrice}>
+                  {'$' +
+                    ((oneTimeAddOn.quantity *
+                      oneTimeAddOneProducts[0].prices[0].unit_amount) /
```

Comment:
The `oneTimeAddOneProducts` (sic) are being referenced by a hard-coded `0` here, so the add-ons list will show all purchased add-ons as being the first in the list (in the picture, I had purchase the 1,200 min NLP add-on):

![image](https://github.com/kobotoolbox/kpi/assets/7819986/881bc926-133b-4b5f-99d1-bfed54d118aa)

You should be able find the right product by matching `oneTimeAddOneProduct.id` with `oneTimeAddOn.product`.